eks/multicam_smoother.py

```diff
@@ -473,6 +376,64 @@ def multicam_smooth_min(smooth_param, cov_matrix, y, m0, S0, C, A, R, ensemble_v
     return nll
 
 
+def mA_compute_maha(scaled_emA_preds, emA_vars, emA_likes,
```

Comment:
does this function have any effect if `inflate_vars` is False? if not, I'd rather the flag check logic occur outside of the function, and then this function can be called something like `inflate_variances_all_keypoints` (and it will always perform that function) to make code clearer